modules/example-apache-httpd-configuration.adoc

```diff
@@ -44,7 +46,7 @@ scripting in Apache HTTPD to accomplish this. This section provides examples.
 #!/bin/bash
 echo "Content-Type: application/json"
 echo ""
-echo '{"sub":"userid", "name":"'$REMOTE_USER'"}'
+echo '{"sub":"'$REMOTE_USER'", "name":"'$REMOTE_USER'"}'
```

Comment:
Your 'oc get user' result copied "Hello basicauth-xxia" as the result of "FULL NAME" column. The result corresponds to how it is defined by "name" here, see bug 1955363#c11 how "name" was correspondingly defined. You missed "Hello " thus your 'oc get user' result does not consist with the definition in line 49.